### PR TITLE
fix: bullet speed notation error

### LINKED ISSUE #43: Fix-bullet-speed-notation-error
## Problem Description

There is an issue where the bullet speed display function is incorrectly shown as the attack speed.

## Todo List

-   [ ] Check the code related to the issue.
-   [ ] Fix code.
-   [ ] Verify fix.

### PR DESCRIPTION
**Pull request by yunsful**
## What
This PR fix bullet speed notation error. It includes the following changes:
Fix bulletspeed getter in ship.class.
Modify the bullet speed draw method in the game screen class.

## Why
There is an issue where the bullet speed display function is incorrectly shown as the attack speed.

## Related Issue
Closes #43 

## How to Test
1. Start the application locally.
2. Verify if the bullet speed value increases correctly when the bullet speed item is collected.

## Screenshots
스크린샷 2024-10-11 오후 5 24 06
스크린샷 2024-10-16 오후 9 01 18

### CODE REVIEW

**Review by LeeHyunWoo02 — APPROVED** [review]
Thank you for your hard work in fixing the sudden bug.
I checked that there is nothing wrong with the code and it works well according to the modifications.

**Review by sml-logs — APPROVED** [review]
LGTM
ScreenShot 2024-10-16 오후 9 31 44

File: src/entity/Ship.java [hunk 1]
```diff
@@ -196,15 +196,16 @@ public void decreaseShootingDelay() {
...
 	 *
-	 * @return Attack speed (bullets per second).
+	 * @return bullet speed (Pixels per frame).
 	 */
-	public final double getAttackSpeed() {
-		return 1000.0 / growth.getShootingDelay();
-	}
+	public final int getBulletSpeed() {
+		int speed = growth.getBulletSpeed();
+		return (speed >= 0) ? speed : -speed;
```
Comment (kojesung) [inline src/entity/Ship.java:207]: It's awesome how you've elegantly used the ternary operator like `(speed >= 0) ? speed : -speed` to address the issues with `Math.abs`.

**Review by kojesung — APPROVED** [review]
image

I have verified that BS is correctly reflected when an item is acquired.

File: src/entity/Ship.java (hunk 1 above)
Comment (DongWonLee2) [reply src/entity/Ship.java:207]: I agree that.

**Review by DongWonLee2 — APPROVED** [review]
It must have been cumbersome, but thank you for reflecting it.

**Review by min5421 — APPROVED** [review]
fix

I checked the bullet speed value increases correctly when the bullet speed item is collected.
Thank you.

**Review by aeioiie — APPROVED** [review]
image
I confirmed that the bullet speed value normally increases when the bullet speed item is obtained. LGTM!

**Pull request merged**